### PR TITLE
Janitor office betterization

### PR DESCRIPTION
**Pull request by patpol4**
### Intent of your Pull Request

Old:
vivaldi_PdOSibhB2q

New:
image

### Why is this change good for the game?
Better organization, no random things on the floor, nice decals, sink

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
see Intent of your Pull Request

### What should players be aware of when it comes to the changes your PR is implementing?
Janitors office looks different

### What general grouping does this PR fall under? 
mapping

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Not really

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
nope

# Changelog

:cl:  
tweak: new janitorial office
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
new roundstart slot for janitor????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

**Comment by patpol4** [comment]
> new roundstart slot for janitor????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

no its just a spawn spot

**Comment by alexkar598** [comment]
IMO the disposal bin and the vapor can should be switched. Its not uncommon to throw things onto tables but here it might go in the disposal bin instead

**Comment by patpol4** [comment]
good idea
